File: example implementations/java/convertor/src/Convert.java
import io.*;
import voxel.VoxModel;

import java.io.File;

public class Convert {
    public static void main(String[] args) {
        File input = new File(args[0]);
        File output = new File(args[1]);
        String inputType = args[0].substring(args[0].lastIndexOf("."));
        if (inputType.equals(".vox"))
            CvoxWriter.write(new CvoxMultimodel(VoxReader.read(input)), output);
        else
            VoxWriter.write(new VoxModel(CvoxReader.read(input)), output);
    }
}

File: example implementations/csharp/cvox-convertor/Convert.cs
using cvox_convertor.io;
using cvox_convertor.voxel;

namespace cvox_convertor
{
    internal class Convert
    {
        public static void Main(string[] args)
        {
            string inputType = args[0].Split('.').Last();
            if (inputType == "vox")
                CvoxWriter.Write(new CvoxMultimodel(VoxReader.read(args[0])), args[1]);
            else
                VoxWriter.Write(new VoxModel(CvoxReader.Read(args[0])), args[1]);
        }
    }
}